lib/pdf_fill/forms/va686c674.rb

```diff
@@ -1586,10 +1638,16 @@ def merge_child_helpers
           # extract ssn
           child['ssn'] = split_ssn(child['ssn'].delete('-')) if child['ssn'].present?
 
-          # extract postal code
+          # extract country: FE uses 3 char country codes, but pdf expects 2 char country code
+          child['place_of_birth']['country'] = extract_country(child['place_of_birth'])
+
+          # extract postal code and country
           unless child['does_child_live_with_you']
             child['child_address_info']['address']['zip_code'] =
               split_postal_code(child.dig('child_address_info', 'address'))
+            # @TODO TEST THIS 20200804
```

Comment:
Does this TODO mean there is still testing required?

Reply:
Related, should we test this postal-code/country split?

Reply:
Thank you!  This comment has been removed.  The split postal code is tested when the filler_spec is run and the overflow.pdf gets generated.  You can see the 9 digit test postal code here: https://github.com/department-of-veterans-affairs/vets-api/blob/d33be24b41da7cc55f0e864697c38b9c7ac08897/spec/fixtures/pdf_fill/686C-674/overflow.json#L201

Reply:
Here is a screenshot of how that postal code is then generated on the PDF:

<img width="881" alt="SplitPostalCode" src="https://user-images.githubusercontent.com/53828142/89942182-f3e87600-dbd0-11ea-844f-47d48675110b.png">